### PR TITLE
Fix the README

### PR DESCRIPTION
**Pull request by JojoVes**
**JIRA Ticket**: [ISLANDORA-2031](https://jira.duraspace.org/browse/ISLANDORA-2031)

# What does this Pull Request do?

Adds useful information to README.md.

# What's new?

* Adds some general organization.
* Adds a requirements section.
* Adds a configuration section.
* Adds a usage section.

# How should this be tested?

Make sure the information is correct and includes all necessary information.

# Interested parties
@Islandora/7-x-1-x-committers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,84 @@
-# ISLANDORA IP EMBARGO
-
-## BUILD STATUS
-Current build status:
-[![Build Status](https://travis-ci.org/Islandora/islandora_ip_embargo.png?branch=7.x)](https://travis-ci.org/Islandora/islandora_ip_embargo)
-
+# Islandora IP Embargo[![Build Status](https://travis-ci.org/Islandora/islandora_ip_embargo.png?branch=7.x)](https://travis-ci.org/Islandora/islandora_ip_embargo)
 CI Server:
 http://jenkins.discoverygarden.ca
```
Comment (qadan) [inline README.md]: This isn't actually a real thing anymore and should be removed

File: README.md [hunk 2]
```diff
@@ -1,32 +1,84 @@
...
+
+## Configuration
+
+### Drupal cron configuration
+
+Drupal cron should be configured to run once a day to clean out expired embargoes.
+
+### Network address lists
+
+Go to `yoursite.com/admin/islandora/tools/ip_embargo` to manage network address lists. You need at least one list to use IP embargo, because these lists are what will appear in the 'IP address range list' drop-down in the 'IP Embargo' tab for any given object, and you can't set an IP embargo without an IP address range list.
```
Comment (qadan) [inline README.md]: Should avoid "you" in READMEs; language like "at least one list is required to use IP embargo" is preferred

Comment (qadan) [reply README.md]: Should do for the rest of instances too

File: README.md [hunk 3]
```diff
@@ -1,32 +1,82 @@
-# ISLANDORA IP EMBARGO
+# Islandora IP Embargo[![Build Status](https://travis-ci.org/Islandora/islandora_ip_embargo.png?branch=7.x)](https://travis-ci.org/Islandora/islandora_ip_embargo)
```
Comment (willtp87) [inline README.md]: Will this work with the module in Islandora-Labs?

File: README.md [hunk 4]
```diff
@@ -1,32 +1,82 @@
...
-A Drupal based approach to embargoing content based on IP ranges.
-This module is not integrated with the Islandora Embargo module.
-This module breaks islandora_xml_site_map with its redirects. 
+Install as usual, see [this](https://drupal.org/documentation/install/modules-themes/modules-7) for further information.
+
+## Configuration
+
+### Drupal cron configuration
+
+Drupal cron should be configured to run once a day to clean out expired embargoes.
```
Comment (willtp87) [inline README.md]: Should probably have `minimally` in there.

File: README.md [hunk 5]
```diff
@@ -1,32 +1,82 @@
-# ISLANDORA IP EMBARGO
+# Islandora IP Embargo[![Build Status](https://travis-ci.org/Islandora/islandora_ip_embargo.png?branch=7.x)](https://travis-ci.org/Islandora/islandora_ip_embargo)
 
-## BUILD STATUS
-Current build status:
-[![Build Status](https://travis-ci.org/Islandora/islandora_ip_embargo.png?branch=7.x)](https://travis-ci.org/Islandora/islandora_ip_embargo)
+A Drupal based approach to embargoing content based on IP ranges.
```
Comment (willtp87) [inline README.md]: Usually under an intro section.

File: README.md [hunk 6]
```diff
@@ -1,32 +1,82 @@
...
+This module is not integrated with the Islandora Embargo module.
+This module breaks islandora_xml_site_map with its redirects.
```
Comment (willtp87) [inline README.md]: Usually under a troubleshooting section.

File: README.md [hunk 7]
```diff
@@ -1,32 +1,82 @@
...
-http://jenkins.discoverygarden.ca
+## Requirements
 
-## CONTENTS OF THIS FILE
+The following Drupal modules are required:
 
- * summary
- * configuration
+* [Islandora](https://github.com/islandora/islandora)
+* [Rules](https://drupal.org/project/rules) (While not a dependency, more functionality can be found by installing this module.)
```
Comment (willtp87) [inline README.md]: Usually under an `Optional Requirements` section.

**Review by willtp87 — APPROVED** [review]

**Pull request merged**